llvm/test/CodeGen/X86/avx512fp16-arith.ll

```diff
@@ -384,7 +376,7 @@ declare <8 x half> @llvm.fabs.v8f16(<8 x half>)
 define <8 x half> @fcopysignv8f16(<8 x half> %x, <8 x half> %y) {
 ; CHECK-LABEL: fcopysignv8f16:
 ; CHECK:       ## %bb.0:
-; CHECK-NEXT:    vpternlogq $228, {{\.?LCPI[0-9]+_[0-9]+}}(%rip){1to2}, %xmm1, %xmm0
```

Comment:
Whats happening here?

Reply:
Looks like canonicalizeBitSelect tends to default to vpternlogq instead of vpternlogd for small types - I'll look at a fix.